packages/jellyfish-api-core/__tests__/category/loan/auctionBid.test.ts

```diff
@@ -0,0 +1,350 @@
+      expect(Number(tslaAmt)).toBeLessThan(30000)
+
+      const promise = tGroup.get(1).rpc.loan.auctionBid({
+        vaultId: vaultId,
+        index: 0,
+        from: node1ColAddr,
+        amount: '30000@TSLA'
+      })
+      await expect(promise).rejects.toThrow(RpcApiError)
+      await expect(promise).rejects.toThrow(`amount ${tslaAmt} is less than 30000.00000000`)
+    })
+
+    it('should not auctionBid as first bid should include liquidation penalty of 5%', async () => {
+      const promise = tGroup.get(1).rpc.loan.auctionBid({
+        vaultId: vaultId,
+        index: 0,
+        from: node1ColAddr,
+        amount: '500@TSLA' // (500 * 5%) + 500 = 525, should not less than 525
+      })
+      await expect(promise).rejects.toThrow(RpcApiError)
+      await expect(promise).rejects.toThrow('First bid should include liquidation penalty of 5%')
+    })
+
+    it('should auctionBid', async () => {
+      const node1ColAccBefore = await tGroup.get(1).rpc.account.getAccount(node1ColAddr)
+      const node1BeforeTSLAAcc = node1ColAccBefore.length > 0
+        ? node1ColAccBefore.find((amt: string) => amt.split('@')[1] === 'TSLA')
+        : undefined
+      const node1BeforeTSLAAmt = node1BeforeTSLAAcc !== undefined ? Number(node1BeforeTSLAAcc.split('@')[0]) : 0
+
+      const txid = await tGroup.get(1).rpc.loan.auctionBid({
+        vaultId: vaultId,
+        index: 0,
+        from: node1ColAddr,
+        amount: '525@TSLA' // (500 * 5%) + 500 = 525, min first bid is 525
+      })
+      expect(typeof txid).toStrictEqual('string')
+      await tGroup.get(1).container.generate(1)
+      await tGroup.waitForSync()
+
+      const auctionsBefore = await tGroup.get(1).container.call('listauctions')
+
+      const node1ColAccAfter = await tGroup.get(1).rpc.account.getAccount(node1ColAddr)
+      const node1AfterTSLAAcc = node1ColAccAfter.length > 0
+        ? node1ColAccAfter.find((amt: string) => amt.split('@')[1] === 'TSLA')
+        : undefined
+      const node1AfterTSLAAmt = node1AfterTSLAAcc !== undefined ? Number(node1AfterTSLAAcc.split('@')[0]) : 0
+      expect(node1BeforeTSLAAmt - node1AfterTSLAAmt).toStrictEqual(525)
+
+      // end the auction and node1 get the auction bid
+      await tGroup.get(0).generate(36)
+      const auctionsAfter = await tGroup.get(1).container.call('listauctions')
+      expect(auctionsBefore[0].batches.length - auctionsAfter[0].batches.length).toStrictEqual(1)
+
+      const node1ColAccAfterWin = await tGroup.get(1).rpc.account.getAccount(node1ColAddr)
+      const wonBids = auctionsBefore[0].batches[0].collaterals
+      expect(node1ColAccAfterWin).toStrictEqual(wonBids.concat(node1ColAccAfter))
+    })
+  })
+
+  describe('test next auctionBid and utxos', () => {
+    beforeAll(async () => {
+      await tGroup.start()
+      await tGroup.get(0).container.waitForWalletCoinbaseMaturity()
+      await setup()
+
+      await tGroup.get(1).rpc.loan.auctionBid({
+        vaultId: vaultId,
```

Comment:
```suggestion
        vaultId,
```

Also apply to the rest of the code

Reply:
hmm.. actually `{ vaultId: vaultId }` or `{ vaultId }` has no diff for me
if we really want go for this styling, i'd fix it in eslint quickly since we're not using `ts-standard`
@fuxingloh